### PR TITLE
Update Rosario's profile

### PR DESCRIPTION
**Pull request by rpintosl**

### CODE REVIEW

File: team/_posts/2020-08-03-pintos-lobo-rosario.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+# social
+cv: https://drive.google.com/file/d/1pOiiG8js0PX_aUk6chkCKFmTO0Xl3gng/view?usp=sharing
+email: rpintosl@fiu.edu
+github: rpintosl
+orcid:
+osf:
+publons:
+researchgate: Rosario_Pintos_Lobo
+scholar: https://scholar.google.com/citations?hl=en&user=gLjcvmkAAAAJ&view_op=list_works&gmla=AJsN-F4S_3SlZ0KhZYLg1M5DKE9LmTFNfUhCM3K_zXN3at0XBy5WVKmPw0RQgfikViKbWNCFH-EvF6zAxr6QcScCYuTV8H-SAQ
```
Comment (tsalo) [inline team/_posts/2020-08-03-pintos-lobo-rosario.md]:
```suggestion
scholar: gLjcvmkAAAAJ
```
Just provide the user ID.

**Review by tsalo — CHANGES_REQUESTED** [review]
There are a few changes that need to be made.
1. The dates in the team filenames don't correspond to when the person joined the lab. They are just used to define the order of the lab members on the Team page. First the PI, then the co-PI, then research associates, then post-docs, then grad students, etc. Your file's date should be "2017-04-09" as a second-year grad student.
2. There is a conflict with your photo. You need to resolve this by choosing one version of the photo over another. You should be able to do it on GitHub Desktop or from the command-line. I would recommend updating from the main repository on your laptop with GitHub Desktop.

File: team/_posts/2017-05-09-pintos-lobo-rosario.md [hunk 2]
```diff
@@ -1,27 +1,26 @@
...
-cv:
+cv: https://drive.google.com/file/d/1pOiiG8js0PX_aUk6chkCKFmTO0Xl3gng/view
 email: rpintosl@fiu.edu
 github: rpintosl
 orcid:
 osf:
 publons:
 researchgate: Rosario_Pintos_Lobo
-scholar:
+scholar: https://scholar.google.com/citations?hl=en&user=gLjcvmkAAAAJ&view_op=list_works&gmla=AJsN-F4S_3SlZ0KhZYLg1M5DKE9LmTFNfUhCM3K_zXN3at0XBy5WVKmPw0RQgfikViKbWNCFH-EvF6zAxr6QcScCYuTV8H-SAQ
```
Comment (tsalo) [inline team/_posts/2017-05-09-pintos-lobo-rosario.md]:
```suggestion
scholar: gLjcvmkAAAAJ
```
Just the ID- not the full URL.

**Review by tsalo — CHANGES_REQUESTED** [review]
There are a bunch of new files that should be removed, like `posters/_posts/.Rhistory` and `team/_posts/.Rhistory`. Also, the folder `team/_posts/_site/` should be removed.

Finally, please look at the suggested change to the Google Scholar link.

**Comment by rpintosl** [comment]
@tsalo I removed the two files (posters/_posts/.Rhistory and team/_posts/.Rhistory, and accepted your suggested change on google scholar. I can't figure out how to remove an entire folder (team/_posts/_site/).

**Comment by rpintosl** [comment]
20171018203609_IMG_0356

**Comment by rpintosl** [comment]
@tsalo I *think* I successfully updated the picture? I committed to rpintosl-patch-2

**Comment by tsalo** [comment]
I'm not seeing it. Are you referring to the comment where you posted the picture above?

**Comment by rpintosl** [comment]
> I'm not seeing it. Are you referring to the comment where you posted the picture above?

@tsalo No, sorry. That was just me checking to see if it worked. I committed from Github desktop but I just saw your message saying to upload directly on Github. Where in Github should I upload?

**Comment by tsalo** [comment]
If you committed in GitHub Desktop you just need to push to GitHub. I'd go with that instead of duplicating effort by adding the file on the website.

**Review by tsalo — APPROVED** [review]
Everything looks great! Thanks!

**Pull request merged**